src/styles/blocks/page.scss

```diff
@@ -0,0 +1,52 @@
+.page {
+  scroll-behavior: smooth;
```

Comment:
Consider adding a fallback for browsers that do not support 'scroll-behavior'. You can use a JavaScript library like 'smoothscroll-polyfill'.